build.py

```diff
@@ -142,7 +142,7 @@ def do_make(target=None, log=False):
 token = None
 job = None
 boot_cmd = None
-use_git = False
+use_git = True
```

Comment:
One more thing, the ``-g`` option is to enable git, so if it's always enabled by default then the option is redundant.  This would make sense when git and the environment are used exclusively, but not when they're both needed (as in my earlier comment).